BTSPocket/ViewModels/TeamsViewModel.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  TeamsViewModel.swift
+//  BTSPocket
+//
+//  Created by bts on 27/04/21.
+//
+
+import Foundation
+
+struct TeamViewModel {
+    //Fuction that comes the data of the team in BTS
+    func getTeamMembers(_ numberOfPage: Int, _ completition: @escaping( (Result<PaginationAllUsers, Error>) -> Void )) {
+        if let userId = BTSApi.shared.currentSession?.id,
+           let token = BTSApi.shared.sessionToken {
```

Comment:
**spaghetti code:** this token should be  placed o as default header in the **WSAPI** implementation